lightning_pose_app/ui/project.py

```diff
@@ -701,27 +631,52 @@ def _render_streamlit_fn(state: AppState):
         )
         state.st_existing_project_format = st_prev_format
 
-        uploaded_file = st.file_uploader(
-            "Upload project in .zip file", type="zip", accept_multiple_files=False)
+        if state.st_existing_project_format in ["DLC", "Lightning Pose"]:
+            uploaded_file = st.file_uploader(
+                "Upload project in .zip file", type="zip", accept_multiple_files=False)
+            if uploaded_file is not None:
+                # read it
+                bytes_data = uploaded_file.read()
+                # name it
+                filename = uploaded_file.name
+                filename_temp = filename.replace(".zip", '_temp.zip')
+                filepath = os.path.join(os.getcwd(), "data", filename_temp)
+                # write the content of the file to the path if it doesn't already exist
+                if not os.path.exists(filepath):
+                    with open(filepath, "wb") as f:
+                        f.write(bytes_data)
+                # check files
+                state.st_error_flag, state.st_error_msg = check_files_in_zipfile(
+                    filepath, project_type=st_prev_format)
+                # grab keypoint names
+                st_keypoints = get_keypoints_from_zipfile(filepath, project_type=st_prev_format)
+                # update relevant vars
+
+                state.st_upload_existing_project_zippath = filepath
+                enter_data = True
+                st_mode = CREATE_STR
+
+        elif state.st_existing_project_format == "SLEAP":
+            uploaded_file = st.file_uploader(
+                "Upload project in .pkg.slp file",
+                type="pkg.slp",
+                accept_multiple_files=False,
+                key="uploader2"
+            )
+
         if uploaded_file is not None:
```

Comment:
this whole `if` block should be indented, otherwise it will run for DLC/LP projects uploads as well, and will cause and error since those will not have the `.pkg.slp` suffix